events/message_poll_events.go

```diff
@@ -0,0 +1,34 @@
+package events
+
+import (
+	"github.com/disgoorg/disgo/discord"
+	"github.com/disgoorg/snowflake/v2"
+)
+
+// GenericMessagePollVote is a generic poll vote event (requires gateway.IntentGuildMessagePolls or gateway.IntentDirectMessagePolls)
+type GenericMessagePollVote struct {
```

Comment:
do we want some more cache getters?
`Channel() (discord.GuildMessageChannel, bool)`
`Member() (discord.Member, bool)`

Reply:
channel sure, member probably not